plugins/net_plugin/net_plugin.cpp

```diff
@@ -2554,6 +2556,7 @@ namespace eosio {
                }
             }
 
+            // use chain head instead of fork head so we do not get too far ahead of applied blocks
```

Comment:
Nothing change here, only a comment added?

Reply:
`get_chain_head_num()` now returns `chain_head` instead of fork head.